### DISCUSSION TITLE
Milvus和JuiceFS的整合还有继续进行和支持吗？

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#38763] [Q&A and General discussion] [answered]
如题。Query Node和JuiceFS整合的话，是会把JuiceFS当成本地磁盘吗？是不是说，QueryNode有可能变成完全无状态的，比如diskann部署在JuiceFS上。QueryNode如果完全无状态，共享存储的话，还是非常有帮助的，scale 的开销应该会小很多，因为避免了shuffle/migrate data？

请问我以上理解对吗？Milvus和JuiceFS的整合还有继续吗？

**Comment by yhmo** [comment, marked as answer]
据我所知，在Milvus v1.x时期有过JuiceFS整合的工作。但在Milvus v2.x之后，就没有继续了。

**Reply by xiaobingxia-at** [reply]
谢谢
